ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -545,7 +545,14 @@ public void setUnderlineColor(ReactEditText view, @Nullable Integer underlineCol
     if (underlineColor == null) {
       drawableToMutate.clearColorFilter();
     } else {
-      drawableToMutate.setColorFilter(underlineColor, PorterDuff.Mode.SRC_IN);
+      if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
Please add a comment here on why this is needed otherwise would be really hard to understand afterwards.